index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Landing</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="container header-container"></div>
+  </header>
+  <main>
+    <div class="hero-section">
+      <div class="container">
+        <h1 class="hero">Account Management Advanced</h1>
```

Comment:
`hero` is no relevant class name for the heading element, better to name it like something like this: `header-heading` or using BEM `header__heading`